lib/landing_session.js

```diff
@@ -20,7 +20,7 @@ class LandingSession extends Session {
     this.startLanding();
     const status = metadata.status ? 'should be ready' : 'is not ready';
     const shouldContinue = await cli.prompt(
-      `This PR ${status} to land, do you want to continue?`);
+      `This PR ${status} to land, do you want to continue?`, metadata.status);
```

Comment:
I wasn't sure about this. Do we want to change the default for this prompt? We *could* only change it if the command is running on non-interactive mode. On the other hand, it kinda makes sense for this prompt to default to false.

Reply:
If it's --yes, then it makes more sense to make set this to true in that mode.

Reply:
In that case it would also make sense to change the default of the last check (which uses `core-validate-commit`) to true. I'm not sure we want that behavior though.

Reply:
I kept this with the previous behavior (default to Y) unless --yes is given, in which case it will default to the result of the metadata check.